src/libmongoc/tests/test-mongoc-interrupt.c

```diff
@@ -106,7 +106,7 @@ test_interrupt (void)
    mock_server_run (server);
    uri = mock_server_get_uri (server);
    stream =
-      mongoc_client_connect_tcp (10000, mongoc_uri_get_hosts (uri), &error);
+      mongoc_client_connect_tcp (NULL, 10000, mongoc_uri_get_hosts (uri), &error);
```

Comment:
I don't think we need to support a custom `bind_ip` for this class.

Reply:
Agreed, mongoc_interrupt can be removed entirely.